help/ant.txt

```diff
@@ -38,6 +38,11 @@ ant idea            => gradlew idea         [better: import as a Gradle project
 Solr-specific targets
 ---------------------
 
+Create a runnable Solr instance
+
+ant server          => gradlew -p solr assemble   [Note that this deletes any previous ./solr/packaging/build/solr-#.#.#-SNAPSHOT directory.]
```

Comment:
Technically, it restores the packaging to its pristine state (syncs rather than deletes/ recreates).